central/main.go

```diff
@@ -312,6 +313,7 @@ func startServices() {
 	pruning.Singleton().Start()
 	gatherer.Singleton().Start()
 	vulnRequestManager.Singleton().Start()
+	phonehome.GathererSingleton().Start()
```

Comment:
Even though the condition is already available as `phonehome.Enabled()`, I think it is not required to be used here thanks to the nil check.
Extracting the nil check from the methods would require all usage places to check for the environment variable. The approach of nil instance in case of disabled telemetry looks safe and transparent to me.